File: source/code/csharp/multitenancy/add_account_to_org.cs
var chewie = client.Instantiate<IAccount>()
    .SetGivenName("Chewbacca")
    .SetSurname("the Wookiee")
    .SetUsername("rrwwgggh")
    .SetEmail("chewie@kashyyyk.rim")
    .SetPassword("Changeme123!");
chewie.CustomData.Put("favoriteShip", "Millennium Falcon");

await bankOfAOrg.CreateAccountAsync(chewie);

File: source/code/java/multitenancy/add_account_to_org.java
Account chewie = client.instantiate(Account.class)
    .setGivenName("Chewbacca")
    .setSurname("the Wookiee")
    .setUsername("rrwwgggh")
    .setEmail("chewie@kashyyyk.rim")
    .setPassword("Changeme123!");
chewie.getCustomData().put("favoriteShip", "Millennium Falcon");

bankOfAOrg.createAccount(chewie);
